app/assets/src/components/common/ProjectCreationForm.jsx

```diff
@@ -14,42 +14,83 @@ import { logAnalyticsEvent } from "~/api/analytics";
 
 import cs from "./project_creation_form.scss";
 
+const ACCESS_LEVEL = Object.freeze({
+  publicAccess: 1,
+  privateAccess: 0,
+  noSelection: -1,
+});
+
 class ProjectCreationForm extends React.Component {
   state = {
     name: "",
-    publicAccess: -1, // No selection by default
+    accessLevel: ACCESS_LEVEL.noSelection, // No selection by default
     error: "",
     description: "",
     showInfo: false,
+    disableCreateButton: true,
   };
 
+  areCreationReqsUnmet(changed = {}) {
```

Comment:
nit: it is normally easier to process when we name it on the positive and then do `!creationReqsMet`...